Codo-Baggins/covid_comparisons
# Feature/comparison card

## What's this PR do?
- Create `ComparisonCard` component
- Create logic and boiler plate text for comparison details page
- Add initial styling to comparison details page
- Add additional state in `ComparisonDetails` to account for both displays of icons

## Where should the reviewer start?
- `ComparisonDetail.tsx` and `ComparisonCard.tsx`

## How should this be manually tested?
- Pick a comparison from the drop down menu and click the button
- Ensure the display shows a different number of 'Xs' for each display
- NOTE: each 'X' represents 100 deaths

## Any background context you want to provide?
- The 'Xs' will be changed to icons on the next styling branch

## Screenshots/GIFs
<img width="1538" alt="Screen Shot 2021-01-12 at 11 52 30 AM" src="https://user-images.githubusercontent.com/68157759/104345790-ba43d000-54cc-11eb-9475-534d8edc17bf.png">
